src/Hint/Extensions.hs

```diff
@@ -456,6 +461,11 @@ used MagicHash = hasS f ||^ hasS isPrimLiteral
     f :: RdrName -> Bool
     f s = "#" `isSuffixOf` occNameStr s
 used PatternSynonyms = hasS isPatSynBind ||^ hasS isPatSynIE
+used ImportQualifiedPost = hasS isImportQualifiedPost
```

Comment:
```suggestion
used ImportQualifiedPost = hasS (== QualifiedPost)
```

Is that sufficient?

Reply:
Well it does hypothetically contain breakage to `ghc-lib-parser-ex`, potentially saving users the hassle.

There are a bunch of other predicates like this  in `ghc-lib-parser-ex` (e.g. [isPViewPat](https://hackage.haskell.org/package/ghc-lib-parser-ex-8.10.0.14/docs/src/Language.Haskell.GhclibParserEx.GHC.Hs.Pat.html#isPViewPat)), I'm guessing with the same intention (?)

Reply:
Yeah the suggestion looks good.
@googleson78 I'm not sure what you mean 🙂 Why is it a breakage?

Reply:
If the `QualifiedPost` constructor ever changes, if `ghc-lib-parser-ex` exposes `isImportQualifiedPost`, only it would have to change, whereas by using `== QualifiedPost` you will also have to change your code.

Reply:
I see. Yeah, `isImportQualifiedPost` will be added to `ghc-lib-parser-ex`.

Reply:
The GHC API is a huge surface area that certainly will change - but not in ways we can predict. All we can do is have tests and write simple code - premature dealing with potential breakages is not that helpful. They always surprise us!